driver/ppm_flag_helpers.h

```diff
@@ -9,8 +9,7 @@ or GPL2.txt for full copies of the license.
 
 #ifndef PPM_FLAG_HELPERS_H_
 #define PPM_FLAG_HELPERS_H_
-
-#include <asm/mman.h>
+#include <linux/mman.h>
```

Comment:
Won't this need a #if .. #else ... #endif too ? 

This is going to fail on Kernels < 5.1.0

Reply:
I tested this on a 5.0 kernel and it compiles fine. 

linux/mman.h was originally checked in for 2.6.12. It has included asm/mman.h from its original commit (https://github.com/torvalds/linux/blob/1da177e4c3f41524e886b7f1b8a0c1fc7321cac2/include/linux/mman.h#L8). This change should be fully backward compatible.

Reply:
Cools!